src/components/uast/Node.js

```diff
@@ -15,7 +15,10 @@ const COLLAPSIBLE_EXTENDED = "'-'";
 export const StyledItem = styled.div`
   margin-left: ${INDENT_SIZE}px;
   min-width: 400px;
-  background: ${props => (props.highlighted ? background.highlight : 'none')};
+  background: ${props =>
+    props.highlighted
+      ? background.highlight
+      : props.hovered ? '#f9f9f9' : 'none'};
```

Comment:
If it's possible, we should use one of the colour of our palette at [variables.js](https://github.com/bblfsh/dashboard/blob/master/src/styling/variables.js)
If it's not possible, let's define a new one there, and use it here

Reply:
okay let's define the new one. We need something very light. Or it will look bad.